<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadixConverter</class>
 <widget class="QWidget" name="RadixConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radix Converter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelTop">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(154, 153, 150);
</string>
       </property>
       <property name="text">
        <string>TextLabel=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelUp">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>35</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEditUp">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxUp">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>246</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEditDown">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxDown">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="labelBottom">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>you should notice something!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="comboBoxTop">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
